app/views/works/index.html.erb

```diff
@@ -0,0 +1,89 @@
+<h2> List of all works</h2>
+
+<section>
+  <h4>Albums</h4>
+<table id='albums-table' class="table table-hover">
+  <thead class="thead-light">
+  <tr>
+    <th>Votes</th>
+    <th>Title</th>
+    <th>Created By</th>
+    <th>Published</th>
+    <th>Upvote</th>
+  </tr>
+  </thead>
+
+  <tbody>
+  <% albums = @works.filter{ |work| work.category == 'album'}%>
+  <% albums.each do |work| %>
+    <tr>
+      <td><%= work.votes.length %></td>
+      <td><%= link_to work.title, work_path(work.id)%></td>
+      <td><%= work.creator %></td>
+      <td><%= work.publication_date %></td>
+      <td><%= link_to "Upvote", vote_path(work_id: work.id), method: :post, class: "btn btn-info" %></td>
+    </tr>
+  <% end %>
+  </tbody>
+</table>
```

Comment:
Note that this loop where you're listing works and votes in several views.  You can dry this up with a partial view that several other views use.  This view for example could use the same partial 3 times and pass in a different collection of works, one for books, one for albums and one for movies.